db/migrate/20151107154547_create_lists.rb

```diff
@@ -7,7 +7,8 @@ def change
       t.timestamps null: false
     end
 
-    ActiveRecord::Base.connection.execute("DELETE from people")
+    ActiveRecord::Base.connection.execute("DELETE from purchased_gifts;
```

Comment:
You can't modify an existing migration. This has been ran on other people's machines. If you need this functionality, you should add it in a new migraiton.